Example/Podfile

```diff
@@ -14,7 +14,7 @@ target 'Emission' do
     pod 'Artsy+UIFonts', :git => "https://github.com/artsy/Artsy-OSSUIFonts.git"
   end
 
-  pod 'React', :path => react_path, :subspecs => ['RCTWebSocket']
+  pod 'React', :path => react_path, :subspecs => ['RCTWebSocket', 'RCTImage']
```

Comment:
By using RN's Image now, we're pulling in a lot of code that we might need to maintain. This might be something we want to do in the future, but for just this image it seems a bit overkill.

I think you may actually be able to solve this issue by wrapping the image view in a wrapper view or otherwise forward the native props and normal props as outlined in that doc.